### PR TITLE
fix: Optional arguments in `get_command` intrinsic

### LINKED ISSUE #4725: `get_command` intrinsic accepts 0 to 3 arguments
```fortran
PROGRAM test_get_command
  CHARACTER(len=255) :: cmd
  CALL get_command(cmd)
  print *, cmd
END PROGRAM
```
```console
$ gfortran a.f90 && ./a.out
 ./a.out                                                                                                                                                                                                                                                        
$ lfortran a.f90
semantic error: Incorrect number of arguments passed to the 'get_command' intrinsic. It accepts exactly 3 arguments.
  --> a.f90:13:3
   |
13 |   CALL get_command(cmd)
   |   ^^^^^^^^^^^^^^^^^^^^^ 


Note: Please report unclear, confusing or incorrect messages as bugs at
https://github.com/lfortran/lfortran/issues.
```

### PR DESCRIPTION
**Pull request by jinangshah21**
Fixes #4725

### CODE REVIEW

**Comment by jinangshah21** [comment]
Here `subroutine_13.f90` is running in linux but giving segfault in mac.

**Comment by Pranavchiku** [comment]
I'm checking

File: src/libasr/pass/intrinsic_subroutines.h [hunk 1]
```diff
@@ -314,26 +323,29 @@ namespace GetCommand {
...
-        dep.push_back(al, s2c(al, c_func_name_1));
-        dep.push_back(al, s2c(al, c_func_name_2));
-        dep.push_back(al, s2c(al, c_func_name_3));
-
         Vec<ASR::expr_t*> call_args; call_args.reserve(al, 0);
-        body.push_back(al, b.Assignment(args[0], b.Call(s_1, call_args, arg_types[0])));
-        body.push_back(al, b.Assignment(args[1], b.Call(s_2, call_args, arg_types[1])));
-        body.push_back(al, b.Assignment(args[2], b.Call(s_3, call_args, arg_types[2])));
+
+        if(arg_types[0]){
```
Comment (Pranavchiku) [inline src/libasr/pass/intrinsic_subroutines.h:328]:
```suggestion
        if(arg_types.size() > 0){
```

File: src/libasr/pass/intrinsic_subroutines.h [hunk 2]
```diff
@@ -314,26 +323,29 @@ namespace GetCommand {
...
+            fill_func_arg_sub("command", arg_types[0], InOut);
+            ASR::symbol_t *s_1 = b.create_c_func_subroutines(c_func_name_1, fn_symtab, 0, arg_types[0]);
+            fn_symtab->add_symbol(c_func_name_1, s_1);
+            dep.push_back(al, s2c(al, c_func_name_1));
+            body.push_back(al, b.Assignment(args[0], b.Call(s_1, call_args, arg_types[0])));
+        }
+        if(arg_types[1]){
```
Comment (Pranavchiku) [inline src/libasr/pass/intrinsic_subroutines.h:335]:
```suggestion
        if(arg_types.size() > 1){
```

File: src/libasr/pass/intrinsic_subroutines.h [hunk 3]
```diff
@@ -314,26 +323,29 @@ namespace GetCommand {
...
+            fill_func_arg_sub("length", arg_types[1], InOut);
+            ASR::symbol_t *s_2 = b.create_c_func_subroutines(c_func_name_2, fn_symtab, 0, arg_types[1]);
+            fn_symtab->add_symbol(c_func_name_2, s_2);
+            dep.push_back(al, s2c(al, c_func_name_2));
+            body.push_back(al, b.Assignment(args[1], b.Call(s_2, call_args, arg_types[1])));
+        }
+        if(arg_types[2]){
```
Comment (Pranavchiku) [inline src/libasr/pass/intrinsic_subroutines.h:342]:
```suggestion
        if(arg_types.size() > 2){
```

**Review by Pranavchiku — APPROVED** [review]

**Comment by jinangshah21** [comment]
Thanks @Pranavchiku !!

**Pull request merged**